### PR TITLE
Replace grunt-force with grunt-force-task

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

**Comment by mightyiam** [comment]
Please review...

File: package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "grunt-contrib-stylus": "~0.5.0",
     "grunt-contrib-uglify": "~0.2.0",
     "grunt-contrib-watch": "~0.5.3",
-    "grunt-force": "~1.0.0",
+    "grunt-force-task": "^1.0.0",
```
Comment (mightyiam) [inline package.json]: This is the correct package.

File: Gruntfile.js [hunk 2]
```diff
@@ -622,7 +624,7 @@ module.exports = function (grunt) {
     grunt.loadNpmTasks("grunt-usemin");
     grunt.loadNpmTasks("grunt-bower-install-simple");
     grunt.loadNpmTasks("grunt-bower-linker");
-    grunt.loadNpmTasks("grunt-force");
+    grunt.loadNpmTasks("grunt-force-task");
```
Comment (mightyiam) [inline Gruntfile.js]: grunt-force is used differently than `force:someTask`.

**Comment by mightyiam** [comment]
Merged in order to release.

**Comment by winhamwr** [comment]
:+1:

**Pull request merged**